packages/build-info/src/frameworks/react-router.ts

```diff
@@ -0,0 +1,40 @@
+import { lt } from 'semver'
+
+import { BaseFramework, Category, DetectedFramework, Framework } from './framework.js'
+
+export class ReactRouter extends BaseFramework implements Framework {
+  readonly id = 'react-router'
+  name = 'React Router'
+  configFiles = ['react-router.config.ts', 'react-router.config.js']
+  npmDependencies = ['react-router']
```

Comment:
Do we want to detect the plugin we built as well?

Reply:
Good question. We've been doing that for some of the frameworks, but I was thinking about it and couldn't come up with any reason for it, so I didn't include it. You definitely need react-router to use react-router; checking for additional dependencies won't help.